File: skills/astra-toolkit/clients/typescript/examples/tables-update-one-udt.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get an existing table
const client = new DataAPIClient();
const database = client.db(process.env.ASTRA_DB_API_ENDPOINT!, {
  token: process.env.ASTRA_DB_APPLICATION_TOKEN!,
});
const table = database.table("**TABLE_NAME**");

// Update a row
(async function () {
  await table.updateOne(
    {
      title: "Chemistry Club",
    },
    {
      $set: {
        president: {
          email: "lisa@example.com",
          user_name: "lisa_m",
        },
        vice_president: {
          email: "tanya@example.com",
          user_name: "tanya_o",
        },
      },
    },
  );
})();

File: skills/astra-toolkit/clients/python/examples/tables-update-one-udt.py
import os
from astrapy import DataAPIClient

# Get an existing table
client = DataAPIClient()
database = client.get_database(
    os.environ["ASTRA_DB_API_ENDPOINT"], token=os.environ["ASTRA_DB_APPLICATION_TOKEN"]
)
table = database.get_table("**TABLE_NAME**")

# Update a row
table.update_one(
    {"title": "Chemistry Club"},
    {
        "$set": {
            "president": {
                "email": "lisa@example.com",
                "user_name": "lisa_m",
            },
            "vice_president": {
                "email": "tanya@example.com",
                "user_name": "tanya_o",
            },
        },
    },
)
